aitelemetry/telemetrywrapper.go

```diff
@@ -27,6 +27,7 @@ const (
 	azurePublicCloudStr              = "AzurePublicCloud"
 	hostNameKey                      = "hostname"
 	defaultTimeout                   = 10
+	maxCloseTimeoutInSeconds         = 30
```

Comment:
I'm divided on whether this should start out as a `time.Duration` as best practice, or should remain an int to match the existing vars. It would be harder to accidentally set a very very small or very very large timeout if we created them all as durations from day zero...

Reply:
I agree. I wasn't sure what is best, but I felt like continuing with the seconds-integer convention to lessen the blast radius for the existing code

Reply:
I am definitely team `time.Duration`, but I agree with keeping it consistent here. I think we need a public awareness campaign that `time.Duration` is the right first choice in many circumstances (it's silly to make the user do time math to come up with 300, when they could just have "5m" in a config, and it would Just Work).